plugins/destination/mongodb/client/client_test.go

```diff
@@ -0,0 +1,26 @@
+package client
+
+import (
+	"os"
+	"testing"
+
+	"github.com/cloudquery/plugin-sdk/plugins/destination"
+)
+
+func getTestConnection() string {
+	testConn := os.Getenv("CQ_DEST_MONGODB_TEST_CONN")
```

Comment:
Maybe we should mention this environment variable in the `CONTRIBUTING.md` file, for if you want to run the tests against a different host / port